pkg/kubecost/asset.go

```diff
@@ -143,9 +143,15 @@ func AssetToExternalAllocation(asset Asset, aggregateBy []string) (*Allocation,
 		// labelName should be derived from the mapping of properties to
 		// label names, unless the aggBy is explicitly a label, in which
 		// case we should pull the label name from the aggBy string.
+		// Unless this matches a special aggregation, as we have that mapping already transformed...
 		labelName := aggBy
+		agglName := aggBy
```

Comment:
How is the use of `agglName` different from `labelName` here? Seems like this could have been achieved by just setting `labelName = v` instead of `agglName = v` on line 153?